src/main/java/edu/greenblitz/robotName/subsystems/swerve/Modules/SwerveModule.java

```diff
@@ -83,21 +80,20 @@ public void stop() {
     public SwerveModuleState getModuleState(){
         return new SwerveModuleState(
                 getCurrentVelocity(),
-                new Rotation2d(getModuleAngle())
+                getModuleAngle()
         );
     }
 
-    public boolean isAtAngle(double targetAngleInRads, double tolerance) {
-        double currentAngleInRads = getModuleAngle();
-        return (currentAngleInRads - targetAngleInRads) %(2*Math.PI) < tolerance
-                || (targetAngleInRads - currentAngleInRads) % (2*Math.PI) < tolerance;
+    public boolean isAtAngle(Rotation2d targetAngle, Rotation2d errorAngleTolerance) {
+        return (getModuleAngle().getRadians() - targetAngle.getRadians()) %(2*Math.PI) < errorAngleTolerance.getRadians()
```

Comment:
Not very readable imo

Reply:
i dont understand how can i make this more readable. do you think that is hould convert this boolean expression to an if-else statement?

Reply:
I mostly think you should split it into multiple lines of variable decelerations so its clearer what you're actually doing.

Reply:
ok.